File: Encapculation.java
import java.util.Scanner;
class Adder{
    private int Total=0;
    public Adder(){

    }
    void add(int price){
        Total = Total + price;
    }
    int getTotalEarning(){
        return Total;
    }
}

class Encapculation{
    public static void main(String[] args){
        Adder obj = new Adder();
        int chocolate = 600,strawberry = 700, mango = 1000;
        int choice, exit =1;
        Scanner sc = new Scanner(System.in);
        while(exit != 0){
            System.out.println("Choose the Cake:\n1.Chocolate\n2.Strawberry\n3.Mango\n4.Get Earning\n");
            choice = sc.nextInt();
            switch (choice){
                case 1:
                    obj.add(chocolate);
                    break;
                case 2:
                    obj.add(strawberry);
                    break;
                case 3:
                    obj.add(mango);
                    break;
                case 4:
                    System.out.println("\nThe total earning is: "+ obj.getTotalEarning());
                    System.out.println();
                    break;
                default:
                    exit = 0;
                    System.out.println("Invalid Input");
            }
        }
        sc.close();
    }
}

File: Encapculation.cpp
#include<iostream>
using namespace std;

class Adder{
    int total=0;
    public:
        Adder(){

        }

        void add(int price){
            total= total+price;
        }

        int getTotalEarning() {
            return total;
        }
};

int main(){
    Adder obj;
    int chocolate = 600, butterscotch= 750, vanilla= 300, choice=0;
    while(choice!=5){
        cout<<"Welcome to cake shop"<<endl;
        cout<<"1.Chocolate"<<endl;
        cout<<"2. Butterscotch"<<endl;
        cout<<"3. Vanilla"<<endl;
        cout<<"4. Get total earning"<<endl;
        cout<<"5. Exit"<<endl;
        cin>>choice;
        switch (choice)
        {
        case 1:
            obj.add(chocolate);
            break;
        
        case 2:
            obj.add(butterscotch);
            break;
        
        case 3:
            obj.add(vanilla);
            break;
        
        case 4:
            cout<<"Total earning is "<<obj.getTotalEarning()<<endl;
            break;

        case 5:
            cout<<"Exiting the program...";
            break;
        
        default:
            break;
        }
    }
    return 0;
}